src/kontrol/__main__.py

```diff
@@ -62,6 +64,14 @@ def main() -> None:
     args = parser.parse_args()
     logging.basicConfig(level=_loglevel(args), format=_LOG_FORMAT)
 
+    expected_k_version = KVersion.parse(f'v{pyk.K_VERSION}')
+    actual_k_version = k_version()
+
+    if expected_k_version != actual_k_version:
```

Comment:
`KVersion` has an optional attribute `git` that is set for a source build. To avoid emitting the warning for all source builds, a custom comparison should be implemented.

Reply:
I do not really understand this comment. Can you please clarify?

Reply:
Ok, so we just need to compare `major`, `minor` and `patch` attributes, is that correct?

Reply:
I think I understood but please check the latest commit! Thanks!